Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -271,6 +374,9 @@ Wiimote::Wiimote(const unsigned int index) : m_index(index), ir_sin(0), ir_cos(1
 
   // swing
   groups.emplace_back(m_swing = new ControllerEmu::Force(_trans("Swing")));
+  groups.emplace_back(m_swing_slow = new ControllerEmu::Force(_trans("SwingSlow")));
+  groups.emplace_back(m_swing_fast = new ControllerEmu::Force(_trans("SwingFast")));
```

Comment:
CamelCase shouldn't be used in user-facing strings in this way.

Reply:
Oh, okay. In that case, you should remove the `_trans` macro instead, since it has no effect if the string isn't going to be displayed in the GUI.